client/src/Components/QuestionsContainer/QuestionsList/AnswersList/AnswersList.jsx

```diff
@@ -85,7 +55,7 @@ const AnswersList = (props) => {
             <Col>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</Col>
```

Comment:
Might want to use `style={{ float: right }}` or something similar? not sure if this is best practice